product/views/InventoryGroup.yaml

```diff
@@ -0,0 +1,83 @@
+#
```

Comment:
Do we need this file if you have `product/views/ManageIQ_Providers_ConfigurationManager_InventoryGroup.yaml`?